packages/jest-cli/src/TestSequencer.js

```diff
@@ -21,6 +24,15 @@ type Cache = {
   [key: string]: [0 | 1, number],
 };
 
+// If a test depends on one of these core modules,
+// we assume that the test may be slower because it is an "integration test"
+// that spawn child processes, accesses the file system, etc.
+const coreModuleWeights = {
```

Comment:
Any ideas on other core modules we could include here?
I just picked some arbitrary modules that came to my mind and they caused quite an improvement,  unfortunately running the benchmark on a few projects takes quite some time so I can't try out all the modules.

Reply:
`https` if `http` is worth it

Reply:
Can we extend this comment with the information on how's `10000` different than `100000`? It's not obvious if the weight is size in bytes or maybe something else?